### DISCUSSION TITLE
How to stop Hunt query that takes a long time

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#5225] [Unsupported Versions] [answered]
Sometimes when doing Hunt queries I increase the date ranges.  Then the next time I log into the system and click Hunt from the submenu of an alert it runs the query and has the previous large date range still set from the last time I used it, which cause the queries to take a really long time, and I have no way to cancel it and adjust.  I just have to wait till it's done and then adjust and run the query again.  Is there a way to cancel a Hunt query? I see the spinning status icon but nothing on the webpage is clickable while it's running the search.  Could a Cancel button be added?

**Comment by dougburks** [comment, marked as answer]
Please see https://github.com/Security-Onion-Solutions/securityonion/issues/4161
